Source/DataSources/GeometryHeightProperty.js

```diff
@@ -0,0 +1,183 @@
+define([
+    '../Core/ApproximateTerrainHeights',
+    '../Core/defined',
+    '../Core/defineProperties',
+    '../Core/Check',
+    '../Core/Event',
+    '../Core/GeometryOffsetAttribute',
+    '../Scene/HeightReference',
+    './createPropertyDescriptor',
+    './Property'
+], function(
+    ApproximateTerrainHeights,
+    defined,
+    defineProperties,
+    Check,
+    Event,
+    GeometryOffsetAttribute,
+    HeightReference,
+    createPropertyDescriptor,
+    Property) {
+    'use strict';
+
+    /**
+     * A {@link Property} which computes height or extrudedHeight for a {@link CorridorGraphics}, {@link EllipseGraphics}, {@link PolygonGraphics} or {@link RectangleGraphics}.
```

Comment:
Perhaps better explain flesh this out to mention that the computed height is in relation to the current terrain visualization and/or specified height reference.  `computes height` doesn't really explain anything without context.